src/config/chains.ts

```diff
@@ -207,9 +208,43 @@ export const NETWORK_METADATA: { [chainId: number]: NetworkMetadata } = {
       symbol: "AVAX",
       decimals: 18,
     },
-    rpcUrls: AVALANCHE_RPC_PROVIDERS,
+    rpcUrls: RPC_PROVIDERS[AVALANCHE],
     blockExplorerUrls: [getExplorerUrl(AVALANCHE)],
   },
+  [AVALANCHE_FUJI_TESTNET]: {
+    chainId: "0x" + AVALANCHE.toString(16),
+    chainName: "Avalanche",
+    nativeCurrency: {
+      name: "AVAX",
+      symbol: "AVAX",
+      decimals: 18,
+    },
+    rpcUrls: RPC_PROVIDERS[AVALANCHE_FUJI_TESTNET],
+    blockExplorerUrls: [getExplorerUrl(AVALANCHE_FUJI_TESTNET)],
+  },
+};
+
+export const CHAIN_ICONS = {
```

Comment:
Added icons config for further usage, didn't change all places where icons are used to not overcomplicate testing now